kernel/pte_flags/src/pte_flags_aarch64.rs

```diff
@@ -220,6 +220,7 @@ impl PteFlagsAarch64 {
         Self::from_bits_truncate(
             Self::NORMAL_MEMORY.bits
             | Self::OUTER_SHAREABLE.bits
+            | Self::PAGE_DESCRIPTOR.bits
```

Comment:
Is this right? I thought we only wanted to set this for upper-level page table entries?

If it is correct, please update the function docs accordingly.

Reply:
No, it's required in the last of the four levels (even if it has no effect at this point). It's optional in the first 3. Qemu ensures it's set in the last level. I'll update the docs.

Reply:
Ok I see, thanks for clarifying. We can't really rely on QEMU to do anything for us or even to perform equally to real hardware. Although perhaps you meant that "QEMU checks (and faults) if it's not set in the last level".

See my other comment about the flags.

Reply:
> QEMU checks (and faults) if it's not set in the last level

that's correct